be/src/olap/tablet.h

```diff
@@ -46,47 +46,27 @@ class TabletMeta;
 
 using TabletSharedPtr = std::shared_ptr<Tablet>;
 
+inline TabletSharedPtr to_tablet(const BaseTabletSharedPtr& base) {
+    if (base->is_memory()) {
+        return TabletSharedPtr();
```

Comment:
incomplete?

Reply:
Why is this incomplete? This utility function for pointer conversion is only used in some internal code, another option is inline them, because, after inlining, some logic is redundant.